website/packages/docs/src/pages/css-extraction-webpack.mdx

```diff
@@ -1,49 +1,145 @@
 ---
 section: 50-Guides
-name: Webpack CSS Extraction
+name: Stylesheet extraction
 ---
 
-# Webpack CSS Extraction
+# Stylesheet Extraction
 
-Extracting to a static style sheet can be enabled with some extra configuration.
-We recommend only extracting when building an application for production.
+Stylesheet extraction is **highly recommended** for all component libraries and applications, so that generated class names can be combined and de-duplicated into one external stylesheet. This allows you to enjoy the decreased stylesheet size that comes with atomic styling.
 
-If you're building a design system or component library and shipping to NPM you shouldn't extract just yet,
-let your consumers do it in their app.
+There are two types of stylesheet extraction:
 
-## Setup
+- Platform stylesheet extraction
+- Parcel / Webpack stylesheet extraction
 
-Before continuing make sure to install the [webpack loader](/installation#webpack).
+## Which type should I use?
 
-### Compiled configuration
+If your package is a component library, use our **platform stylesheet extraction**. This includes Atlassian Design System packages like `@atlaskit/button`; or any other package that isn't run directly, but is imported and used by _another codebase_ (e.g. Jira, Confluence).
```

Comment:
Do we want to say "Jira, Confluence" here?  I'm fine with it, but I would probably use more generic wording within the design system:
```suggestion
If your package is a component library, use our **platform stylesheet extraction**. This includes Atlassian Design System packages like `@atlaskit/button` or any other package that is treated as a platform offering or used across products or repositories.
```

Reply:
ye sure

i went with giving specific atlassian examples because i anticipate that mostly atlassian devs would be reading these docs

but your wording is better